src/init/features/dataconnect/sdk.ts

```diff
@@ -68,17 +68,8 @@ async function askQuestions(setup: Setup, config: Config): Promise<SDKInfo> {
   }
 
   // First, lets check if we are in an app directory
-  let appDir = process.env[FDC_APP_FOLDER] || process.cwd();
+  const appDir = process.env[FDC_APP_FOLDER] || process.cwd();
   let targetPlatform = await getPlatformFromFolder(appDir);
-  if (targetPlatform === Platform.NONE && !process.env[FDC_APP_FOLDER]?.length) {
```

Comment:
I'm not sure that we can just assume that wherever the firebase.json is, that that's where the user's app is going to be as well